bundles/best_practices.rst

```diff
@@ -166,6 +166,72 @@ the ``Tests/`` directory. Tests should follow the following principles:
    A test suite must not contain ``AllTests.php`` scripts, but must rely on the
    existence of a ``phpunit.xml.dist`` file.
 
+Travis
+------
+
+A popular way to test open source bundles is by using `Travis CI`_. A good practice
+is to support at least the two latest LTS versions of Symfony. One should also test
+test latest beta release. Here is a recommended configuration file (``.travis.yml``).
+
+.. code-block:: yaml
+
+    language: php
+    sudo: false
+    cache:
+        directories:
+            - $HOME/.composer/cache/files
+            - $HOME/symfony-bridge/.phpunit
+
+    env:
+        global:
+            - TEST_COMMAND="./vendor/bin/simple-phpunit"
+            - SYMFONY_PHPUNIT_DIR="$HOME/symfony-bridge/.phpunit"
+
+    matrix:
+        fast_finish: true
+        include:
+              # Minimum supported Symfony version with the latest PHP version
+            - env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" SYMFONY_DEPRECATIONS_HELPER="weak"
+
+              # Test the latest stable release
+            - php: 7.0
+            - php: 7.1
+            - php: 7.2
+              env: COVERAGE=true TEST_COMMAND="./vendor/bin/simple-phpunit --coverage-text"
+
+              # Test LTS versions. This makes sure we do not use Symfony packages with version greater
+              # than 2 or 3 respectively. Read more at https://github.com/symfony/lts
+            - php: 7.2
+              env: DEPENDENCIES="symfony/lts:^2"
+            - php: 7.2
+              env: DEPENDENCIES="symfony/lts:^3"
+
+              # Latest commit to master
+            - env: STABILITY="dev"
+
+        allow_failures:
+              # Dev-master is allowed to fail.
+            - env: STABILITY="dev"
+
+    before_install:
+        - composer require --no-update "symfony/phpunit-bridge:^3.3 || ^4"
+        - if [[ $COVERAGE != true ]]; then phpenv config-rm xdebug.ini || true; fi
```

Comment:
purely personal as well but I find more readable:

```yaml
before_install:
    - |
      if [[ $COVERAGE != true ]];
           then phpenv config-rm xdebug.ini || true;
      fi
```

Reply:
I think this is less readable in the Travis log, as the folding would hide the actual command being run (as it keeps only the first line) AFAIK

Reply:
That's true but at least you can unfold it to see what is being executed as opposed to have a multiple line long command (this one is relatively simple so I wouldn't argue too much about that peculiar example) which remains unreadable in both the config file and on Travis.